media-libs/speex/speex-1.2.0.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit autotools flag-o-matic multilib-minimal
+
+MY_P=${P/_} ; MY_P=${MY_P/_p/.}
+
+DESCRIPTION="Audio compression format designed for speech"
+HOMEPAGE="http://www.speex.org/"
+SRC_URI="http://downloads.xiph.org/releases/speex/${MY_P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ia64 ~mips ~ppc ~ppc64 ~sh ~x86 ~amd64-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~x86-solaris"
+IUSE="dsp fixed-point ogg static-libs +vbr armv4 armv5te cpu_flags_x86_sse"
+
+REQUIRED_USE="dsp? ( ogg )"
+
+RDEPEND="
+	dsp? ( media-libs/speexdsp )
+	ogg? ( media-libs/libogg:= )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+S=${WORKDIR}/${MY_P}
+
+PATCHES=( "${FILESDIR}/${P}-configure.patch" )
+
+src_prepare() {
+	default
+
+	sed -i \
+		-e 's:noinst_PROGRAMS:check_PROGRAMS:' \
+		libspeex/Makefile.am || die
+
+	eautoreconf
+}
+
+multilib_src_configure() {
+	append-lfs-flags
+
+	ECONF_SOURCE="${S}" econf \
+		$(use_enable static-libs static) \
+		$(use_enable armv4 arm4-asm) \
+		$(use_enable armv5te arm5e-asm) \
+		$(use_enable cpu_flags_x86_sse sse) \
+		$(use_enable fixed-point) \
+		$(use_enable vbr) \
+		$(use_with dsp speexdsp) \
+		$(use_enable ogg binaries)
+}
+
+multilib_src_install() {
+	emake DESTDIR="${D}" docdir="${EPREFIX}/usr/share/doc/${PF}" install
+
+	prune_libtool_files
```

Comment:
don't use `prune_libtool_files` (for which you're missing `eutils` anyway), just use find: https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Handling_Libtool_Archives
If you fix the `Makefile.am`'s, I would call this in `multilib_src_install_all` instead